app/models/job_posting.rb

```diff
@@ -1,3 +1,13 @@
 class JobPosting < ApplicationRecord
   belongs_to :company_profile
+  belongs_to :job_type
+  belongs_to :experience_level
+  has_rich_text :description
+
+  enum :salary_currency, { usd: "USD", eur: "EUR", brl: "BRL" }
```

Comment:
Even that it's possible to use  a `string` as the valeu for each ENUM key, the result are slower queries in the future (you can check this on the ENUM docs https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)

Let's use integer values so we don't have performance issues, ok?